app/mailers/sign_up_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class SignUpMailer < ApplicationMailer
+  def sign_up_email(user)
+    @user = user
+    mail(to: @user.email, subject: 'Welcome to Droply')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.